src/transactions/dto/transaction-hash-params.dto.ts

```diff
@@ -0,0 +1,8 @@
+/* istanbul ignore file */
+
+import { IsHexadecimal } from 'class-validator';
+
+export class TransactionHashParamsDto {
+  @IsHexadecimal()
```

Comment:
We can do a length check here as well since the transaction hash should be a fixed size.

Reply:
@prashantasdeveloper could you add the length check so we can merge it as well?